src/Runner.Listener/MessageListener.cs

```diff
@@ -74,6 +92,25 @@ public async Task<Boolean> CreateSessionAsync(CancellationToken token)
             string errorMessage = string.Empty;
             bool encounteringError = false;
 
+            var v1Creds = _configStore.GetCredentials();
+            var v2Creds = _configStore.GetV2Credentials();
+            if (v1Creds != null)
+            {
+                _v1CredentialsExists = true;
+            }
+
+            if (v2Creds == null)
+            {
+                if (v1Creds.Scheme == Constants.Configuration.OAuth)
+                {
+                    _needToCheckAuthorizationUrlUpdate = true;
```

Comment:
Hosted runner use OAuthAccessToken instead of OAuth, and we don't need any migration for hosted runner.